exchanges/bybit/bybit_test.go

```diff
@@ -3309,3 +3324,81 @@ func TestGetUSDCPredictedFundingRate(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestUpdateTickers(t *testing.T) {
+	t.Parallel()
+	supportedAssets := b.GetAssetTypes(false)
+	ctx := context.Background()
+	for x := range supportedAssets {
+		err := b.UpdateTickers(ctx, supportedAssets[x])
+		if err != nil {
+			t.Fatalf("%v %v\n", supportedAssets[x], err)
+		}
+
+		avail, err := b.GetAvailablePairs(supportedAssets[x])
+		if err != nil {
+			t.Fatalf("%v %v\n", supportedAssets[x], err)
+		}
+
+		for y := range avail {
+			_, err = ticker.GetTicker(b.GetName(), avail[y], supportedAssets[x])
+			if err != nil {
+				t.Fatalf("%v %v %v\n", avail[y], supportedAssets[x], err)
+			}
+		}
+	}
+}
+
+func TestGetTickersV5(t *testing.T) {
+	t.Parallel()
+	_, err := b.GetTickersV5(context.Background(), "spot", "")
+	if err != nil {
+		t.Error(err)
+	}
+
+	_, err = b.GetTickersV5(context.Background(), "spot", "BTCUSDT")
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
"option" doesn't work yet without baseCoin, would be best to test all categories, but I won't force it 😄